gradle-plugin-testing/src/test/groovy/com/palantir/gradle/plugintesting/PluginTestingPluginIntegrationSpec.groovy

```diff
@@ -70,6 +78,7 @@ class PluginTestingPluginIntegrationSpec extends IntegrationSpec {
                             }
                             dependencies {
                                 // This version causes deprecation warnings in gradle 8 for gradle 9
+                                // DO NOT REPLACE THIS VERSION WITH A 'resolve' call
```

Comment:
can you explain why?

Reply:
I revised this to make it easier to understand and not rely on that specific version of the plugin.  I just inserted a task that spit out the string that nebula test looks for.